react/components/EXPERIMENTAL_Conditions/Statement.js

```diff
@@ -169,9 +169,11 @@ Statement.propTypes = {
   /** Current selected options for this Statement */
   statements: PropTypes.arrayOf(
     PropTypes.shape({
-      subject: PropTypes.string,
-      verb: PropTypes.string,
-      object: PropTypes.any,
+      refs: PropTypes.shape({
```

Comment:
Why that changed? Was it wrong before?

Reply:
Yep. You can see if you open the whole file that we were using it that way. Accessing a `refs` property.